src/dispatch/static/dispatch/src/incident/Table.vue

```diff
@@ -39,6 +39,7 @@
             :sort-by.sync="sortBy"
             :sort-desc.sync="descending"
             :loading="loading"
+            data-testid="incident-data-table"
```

Comment:
We may want to consider a single SOT for these `data-testid` strings as an `Enum` that we can use here and in tests `Pages`